app/driveshare.html

```diff
@@ -308,6 +258,8 @@ <h4 class="modal-title">Console Logs</h4>
 	.script("lib/electron_boilerplate/env_config.js")
 	.script("lib/electron_boilerplate/context_menu.js")
 	.script("lib/electron_boilerplate/external_links.js")
+	/*.script('bower_components/angular/angular.js')
```

Comment:
You should make the changes, commit them to your branch and push them to your fork of the repo, it should automatically update this pull request.